packages/dev-env/docker-compose.yaml

```diff
@@ -3,6 +3,6 @@ version: "3.98"
 services:
   dwn-server:
     container_name: dwn-server
-    image: ghcr.io/tbd54566975/dwn-server:dwn-sdk-0.2.10
+    image: ghcr.io/tbd54566975/dwn-server:dwn-sdk-0.2.16
```

Comment:
```suggestion
    image: ghcr.io/tbd54566975/dwn-server:dwn-sdk-0.2.18
```

Reply:
Addressed in [5167208](https://github.com/TBD54566975/web5-js/pull/445/commits/516720821bcb2dccec9d326823e571bd81af7ea8)